toobusy.js

```diff
@@ -147,12 +147,13 @@ toobusy.onLag = function (fn, threshold) {
  */
 function start() {
   checkInterval = setInterval(function(){
-    var now = Date.now();
-    var lag = now - lastTime;
+    // using hrtime instead of using the machine's datetime to avoid clock drifts.
+    var hrLag = process.hrtime(lastTime);
+    var lag = hrLag[1] / 1000000; // Get the lag in milliseconds.
```

Comment:
This is wrong. `hrLag[0]` contains seconds, which can't be ignored, especially since the regular interval is 500ms.

Reply:
right, this should be fixed.